apis/python/test/test_ingestion.py

```diff
@@ -512,6 +512,18 @@ def test_ingestion_with_updates(tmp_path):
             source_uri=os.path.join(dataset_dir, "data.u8bin"),
             partitions=partitions,
         )
+
+        # TODO(paris): Fix Vamana to have same metadata as Python and re-enable.
```

Comment:
Do we have a story for this? (and pls cross-link)

Reply:
This is addressed in the follow-up here: #359